src/components/themed/ManageTokensRow.js

```diff
@@ -33,7 +33,7 @@ export function ManageTokensRow(props: Props) {
   const isEditable = metaTokens.every(token => token.currencyCode !== currencyCode)
 
   const onPress = () => {
-    isEditable ? goToEditTokenScene(currencyCode) : (() => {})()
+    if (isEditable) goToEditTokenScene(currencyCode)
```

Comment:
This should have been done in `fixup! Remove a few unused or unneeded util functions`, but I'll take it.